app/grants/management/commands/estimate_clr.py

```diff
@@ -47,10 +47,10 @@ def handle(self, *args, **options):
         what = options['what']
         print (network, clr_pk, what)
 
-        if clr_pk == "all":
-            active_clr_rounds = GrantCLR.objects.filter(is_active=True)
-        else:
+        if clr_pk.isdigit():
             active_clr_rounds = GrantCLR.objects.filter(pk=clr_pk)
```

Comment:
why not `GrantCLR.objects.filter(pk=clr_pk, is_active=True)` ?

Reply:
cause we'd want to have the ability to run clr on a round even if it's not active manually to calculate the curve after round ends (aka before payout)